cmake/BLTOptions.cmake

```diff
@@ -61,9 +61,17 @@ option(ENABLE_MPI          "Enables MPI support" OFF)
 option(ENABLE_OPENMP       "Enables OpenMP compiler support" OFF)
 option(ENABLE_CUDA         "Enable CUDA support" OFF)
 
-# Controls if Google Mock is built and available for use. 
-# If ENABLE_TESTS=OFF, no testing support is built and this option is ignored.
+# Options that control if Google Test, Google Mock, and Fruit are built 
+# and available for use. 
+#
+# If ENABLE_TESTS=OFF, no testing support is built and these option are ignored.
+#
+# Google Mock requires and always builds Google Test, so ENABLE_GMOCK=ON implies ENABLE_GTEST=ON.
+#
+option(ENABLE_GTEST        "Enable Google Test testing support (if ENABLE_TESTS=ON)" ON)
 option(ENABLE_GMOCK        "Enable Google Mock testing support (if ENABLE_TESTS=ON)" OFF)
+option(ENABLE_FRUIT        "Enable Fruit testing support (if ENABLE_TESTS=ON and ENABLE_FORTRAN=ON)" ON)
```

Comment:
Do you check if they enable fruit but don't enable fortran?  Same with enable gtest/gmock/fruit without enabling test... I would rather error and exit then ignore an option that they probably got wrong.

Reply:
reviewed with @white238, the fact that these are guarded in a nested manner ( fruit support requires ENABLE_FORTRAN and ENABLE_FRUIT) means we shouldn't hit any strange cases.